promql/PromQLLexer.g4

```diff
@@ -160,7 +164,7 @@ TIME_RANGE
 
 DURATION: [0-9]+ ('s' | 'm' | 'h' | 'd' | 'w' | 'y');
 
-METRIC_NAME: [a-zA-Z_:] [a-zA-Z0-9_:]*;
-LABEL_NAME:  [a-zA-Z_] [a-zA-Z0-9_]*;
+METRIC_NAME options { caseInsensitive=false; }: [a-zA-Z_:] [a-zA-Z0-9_:]*;
+LABEL_NAME options { caseInsensitive=false; }:  [a-zA-Z_] [a-zA-Z0-9_]*;
 
```

Comment:
It can be rewritten without using of option, ANTLR handles ranges as well:

```suggestion
METRIC_NAME: [a-z_:] [a-z0-9_:]*;
LABEL_NAME:  [a-z_] [a-z0-9_]*;
```

Reply:
Nice, much cleaner.